gdpr/utils.py

```diff
@@ -102,6 +122,18 @@ def get_all_obj_and_parent_versions_queryset_list(obj: Model) -> List[QuerySet]:
     return [get_reversion_versions(i) for i in get_all_parent_objects(obj)] + [get_reversion_versions(obj)]
 
 
+def get_all_obj_entries_queryset_list(obj: Model) -> List[QuerySet]:
+    """Return list of object and its parent version querysets"""
+    from gdpr.utils import get_auditlog_entries
+
+    return [get_auditlog_entries(obj)]
+
+
 def get_all_obj_and_parent_versions(obj: Model) -> List[Model]:
     """Return list of all object and its parent versions"""
     return [item for sublist in get_all_obj_and_parent_versions_queryset_list(obj) for item in sublist]
+
+
+def get_all_obj_entries(obj: Model) -> List[Model]:
+    """Return list of all object and its parent versions"""
```

Comment:
this docstring is not up to date.

Also, I'm not quite sure what this function does. Can you elaborate? Why it can't just return:

```
return list(get_all_obj_entries_queryset_list(obj))
```

Reply:
you are right, it can be simplified